modin/experimental/pandas/test/test_io_exp.py

```diff
@@ -119,6 +119,12 @@ def test_read_csv_without_glob(self):
             with pytest.raises(FileNotFoundError):
                 pd.read_csv_glob("s3://nyc-tlc/trip data/yellow_tripdata_2020-")
 
+    # This addresses https://github.com/modin-project/modin/issues/4056
+    def test_read_single_csv_with_invalid_parse_dates(self):
+        modin_df = pd.read_csv_glob(pytest.files[0], parse_dates=[])
+        pandas_df = pandas.read_csv(pytest.files[0], parse_dates=[])
```

Comment:
Should we add the `nonexistent_column` test below here as well?

Reply:
Done.